lib/stats.js

```diff
@@ -50,6 +52,15 @@ function get (pids, options, callback) {
   if (platform !== 'win' && options.usePs === true) {
     fn = ps
   }
+  if (platform === 'win') {
+    try {
+      spawn('wmic', function (err) {
+        if (err) throw new Error(err)
+      })
+    } catch (err) {
+      fn = gwmi
+    }
```

Comment:
I don't like this, isn't there another way to try and see if wmic is installed?

Reply:
Not sure if this PR still wants to be pursued, but leaving a comment in any case:

On Windows there's the `where` [^1] command (~ equivalent to UNIX `which`). This could be used to check whether `gwmi` / `wmic` is available.
Though, this still requires a `spawn` call.  

[^1] https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/where

Reply:
I'd like to be able to continue, but I haven't been following this for a long time and have been a little busy lately with my new job, so I may tackle this again in the next two weeks!